index.js

```diff
@@ -316,3 +345,4 @@ JUnitXmlPlugin.prototype.teardown = async function () {
 
 module.exports = new JUnitXmlPlugin();
 module.exports.JUnitXrayPlugin = JUnitXmlPlugin;
+module.exports.findXrayIdAndName = findXrayIdAndName;
```

Comment:
Do we need to export this?

Reply:
I moved it to a `common.js` file and imported it in the plugin and unit tests.